README.md

```diff
@@ -220,7 +220,7 @@ $ npm install nanostate
 
 ## See Also
 - [Infinitely Better UIs With Finite Automata (video)](https://www.youtube.com/watch?v=VU1NKX6Qkxc&feature=youtu.be)
-- [choojs/fsm-event](https://github.com/choojs/fsm-event)
+- [choojs/fsm-event](https://github.com/yoshuawuyts/fsm-event)
```

Comment:
the link label too!

Reply:
ohhh yeah. this is what happens when you quickly make a change in github's editor 📦